src/antares/service/local_services/link_local.py

```diff
@@ -33,8 +33,8 @@ def __init__(self, config: LocalConfiguration, study_name: str, **kwargs: Any) -
 
     def create_link(
         self,
-        area_from: Area,
-        area_to: Area,
+        area_from: str,
+        area_to: str,
         properties: Optional[LinkProperties] = None,
         ui: Optional[LinkUi] = None,
         existing_areas: Optional[MappingProxyType[str, Area]] = None,
```

Comment:
I didn't see Sigurd introduced this to ensure validation inside local. Not a huge fan at all. We should do this inside the implem and remove this. But we can do this inside another PR

Reply:
PR to come